src/App.tsx

```diff
@@ -9,18 +10,32 @@ import { useStore } from 'store';
 import { Toaster } from 'components/common/Toaster/Toaster';
 import { Footer } from 'components/common/Footer/Footer';
 
-let didInit = false;
+let didInitCheck = false;
```

Comment:
Out of curiosity, is there a reason to put it here over in a state ?

Reply:
with react strict mode disabled, the effect would run twice when the variable is in state.